litmus-portal/backend/subscriber/pkg/cluster/k8s.go

```diff
@@ -1,52 +1,64 @@
-package k8s
+package cluster
```

Comment:
k8.go is moved from k8s pkg to cluster pkg because to reuse type.go structs which are inside cluster pkg.